lib/assertions.js

```diff
@@ -120,6 +120,12 @@ module.exports = function (expect) {
         expect(subject instanceof Constructor, '[not] to be truthy');
     });
 
+    expect.addAssertion('<any> [not] to be one of <any>', function (expect, subject, superset) {
```

Comment:
Should be `<any> [not] to be one of <array>`, otherwise you'll get "superset.indexOf is not a function" when trying with other RHS types.